src/cfnlint/rules/functions/GetAtt.py

```diff
@@ -80,6 +80,15 @@ def match(self, cfn):
             if resname:
                 if resname in valid_getatts:
                     if restype is not None:
+                        # Check for maps
+                        restypeparts = restype.split('.')
+                        if restypeparts[0] in valid_getatts[resname] and len(restypeparts) >= 2:
+                            if valid_getatts[resname][restypeparts[0]].get('Type') != 'Map':
```

Comment:
@kddejong [`AWS::ServiceCatalog::CloudFormationProvisionedProduct`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-servicecatalog-cloudformationprovisionedproduct.html#aws-resource-servicecatalog-cloudformationprovisionedproduct-return-values)'s the only `Map` so far:

```bash
curl -s --compressed https://d1uauaxba7bl26.cloudfront.net/latest/gzip/CloudFormationResourceSpecification.json | jq '.ResourceTypes' | jq 'with_entries(.value |= .Attributes)' | grep -v ': null' | grep -Eo 'Type": ".*"' | sort | uniq -c | sort -nr
 504 Type": "String"
  26 Type": "List"
   6 Type": "Integer"
   4 Type": "Boolean"
   1 Type": "Map"
   1 Type": "Json"
   1 Type": "FindingsFilterListItem"
```

so I was considering hardcoding that exception, but this seems even better 👍

Reply:
Yea, I'm not a huge fan of this solution.  I think there are better long term solutions if this continues to grow.